plutus-ir/test/Lib.hs

```diff
@@ -0,0 +1,87 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Lib where
```

Comment:
Can we name this something more precise?

Reply:
Sure.